bin/taste-tester

```diff
@@ -409,6 +409,16 @@ MODES:
     ) do
       options[:skip_post_test_hook] = true
     end
+
+    opts.on(
+      '--parallel-host-requests NUMHOSTS', 'Start NUMHOSTS threads in parallel'
+    ) do |n|
+      unless n.to_i > 0
+        logger.error("Sorry, number of threads cannot be less than 1: #{n}")
+        exit(1)
+      end
+      options[:parallel_hosts] = n.to_i
```

Comment:
oh onemore comment, please default this to like... 3 or 5 or something small.

Reply:
I think I defaulted it to the number of cores, like what parallel does. I can put that lower.